apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -210,7 +210,7 @@ export class ExtractorConfig {
     path.join(__dirname, '../schemas/api-extractor-defaults.json')
   );
 
-  private static readonly _declarationFileExtensionRegExp: RegExp = /\.d\.ts$/i;
```

Comment:
TypeScript 5 is also adding the ability to reference `.d.*.ts` for typings for non-js files, with `.d.js.ts` technically a valid alternative to standard `.d.ts`. We may want to add that consideration here as well.

Reply:
Do you want me to do that?

Reply:
We can deal with that separately.